config.go

```diff
@@ -0,0 +1,66 @@
+package scopeagent
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"runtime"
+)
+
+type Config struct {
+	CurrentProfile	string				`json:"currentProfile"`
+	Profiles		map[string]Profile	`json:"profiles"`
+}
+
+type Profile struct {
+	ApiEndpoint		string  `json:"apiEndpoint"`
+	ApiKey			string	`json:"apiKey"`
+	OAuthToken		string	`json:"oauthToken"`
+}
+
+func GetConfig() *Config {
+	homeDir, _ := homeDir()
+	filePath := fmt.Sprintf("%s/.scope/config.json", homeDir)
+	file, err := os.Open(filePath)
+	if err != nil {
+		return nil
+	}
+	defer file.Close()
+	fileBytes, _ := ioutil.ReadAll(file)
+	var config Config
+	if err = json.Unmarshal(fileBytes, &config); err != nil {
+		return nil
+	}
+	return &config
+}
+
+func GetConfigCurrentProfile() *Profile {
+	if config := GetConfig(); config != nil && config.Profiles != nil && config.CurrentProfile != "" {
+		profile := config.Profiles[config.CurrentProfile]
+		return &profile
+	}
+	return nil
+}
+
+
+func homeDir() (string, error) {
```

Comment:
Is this the best way to get the home folder? What about

```go
usr, _ := user.Current()
dir := usr.HomeDir
```

Reply:
I have to check if is compatible with Go 1.11, because I was using `os.UserHomeDir()` but is only for Go 1.12.

Reply:
It works in Go 1.11, I've made the change.